### DISCUSSION TITLE
Webhook.edit() fires a WEBHOOKS_UPDATE event twice

### DISCUSSION THREAD

**Discussion by damian-kos** [Rapptz/discord.py#9377] [Q&A] [answered]
So I have this thing which I struggle for couple days now. I know that it can be done so that will fire this event only once `NabBot` for example does exactly that. It edits a current webhook by changing it's channel and it fires only once. However when I try to do so, it fires it twice.
```
intents = discord.Intents.default()
intents.message_content = True
intents = discord.Intents.default()
intents.webhooks = True

bot = commands.Bot(command_prefix='/', intents=intents)

@bot.event
async def on_ready():
    print(f'Logged in as {bot.user.name}')

@bot.event
async def on_message(ctx):
    new_webhook_name ="Test Webhook"
    new_channel = None
    for channel in ctx.guild.channels:
        if channel.name == "search":
            new_channel = channel
    webhooks = await ctx.guild.webhooks()
    for web in webhooks:
        if web.name == "Test Webhook":
            await web.edit(channel=new_channel)
async def on_socket_event_type(event_type):
    print(event_type)
bot.add_listener(on_socket_event_type)

bot.run(TOKEN)
```
results in 
```
MESSAGE_CREATE
WEBHOOKS_UPDATE
WEBHOOKS_UPDATE
```

**Comment by Rapptz** [comment, marked as answer]
This looks like a bug in the library where it was sending two requests when you edit a channel. It's been fixed in https://github.com/Rapptz/discord.py/commit/da10065c1912996ff69ccd21602828edf843f305

**Reply by damian-kos** [reply]
Sorry, but I don't fully understand this then. I'm using `discord 2.2.2` and `discord.py  2.2.2`. How can I make it work on my side? I'm still beginner.

**Reply by Rapptz** [reply]
You can wait for the next update (TBD) or you can install from git. It's better to wait since there's less chance for something to go wrong and break.

**Reply by damian-kos** [reply]
I see, thank you
